proxmoxtf/resource/vm/vm.go

```diff
@@ -4896,6 +4896,90 @@ func vmReadCustom(
 	e = d.Set(mkNodeName, nodeName)
 	diags = append(diags, diag.FromErr(e)...)
 
+	if _, ok := d.GetOk(mkMigrate); !ok {
+		err := d.Set(mkMigrate, dvMigrate)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutClone); !ok {
+		err := d.Set(mkTimeoutClone, dvTimeoutClone)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutCreate); !ok {
+		err := d.Set(mkTimeoutCreate, dvTimeoutCreate)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutMigrate); !ok {
+		err := d.Set(mkTimeoutMigrate, dvTimeoutMigrate)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutReboot); !ok {
+		err := d.Set(mkTimeoutReboot, dvTimeoutReboot)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutShutdownVM); !ok {
+		err := d.Set(mkTimeoutShutdownVM, dvTimeoutShutdownVM)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutStartVM); !ok {
+		err := d.Set(mkTimeoutStartVM, dvTimeoutStartVM)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutStopVM); !ok {
+		err := d.Set(mkTimeoutStopVM, dvTimeoutStopVM)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkTimeoutMoveDisk); !ok {
+		err := d.Set(mkTimeoutMoveDisk, dvTimeoutMoveDisk)
+		if err != nil {
+			diags = append(diags, diag.FromErr(e)...)
+		}
+	}
+
+	if _, ok := d.GetOk(mkStopOnDestroy); !ok {
```

Comment:
for boolean fields we should be using `GetOkExists`. It's deprecated, but essentially we don't want to overwrite, e.g. with false if a value is not defined in the config.